### PR TITLE
Add BigQuery sample

### PR DESCRIPTION
**Pull request by zoercai**
Sample for change stream connector to BigQuery using BigQueryIO, as well as a script to run the sample.

### CODE REVIEW

File: run_bigquery.sh [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    --metadataInstance=${METADATA_INSTANCE} \
+    --metadataDatabase=${METADATA_DATABASE} \
+    --changeStreamName=${CHANGE_STREAM_NAME} \
+    --gcsBucket=${GCS_BUCKET} \
+    --gcpTempLocation=gs://${GCS_BUCKET}/temp \
+    --region=${REGION} \
+    --bigQueryDataset=${BIG_QUERY_DATASET} \
+    --bigQueryTableName=${BIG_QUERY_TABLE_NAME} \
+    --runner=DataflowRunner \
+    --numWorkers=100 \
```
Comment (zoercai) [inline run_bigquery.sh]: change to 1

File: run_bigquery.sh [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    --maxNumWorkers=100 \
+    --experiments=use_unified_worker,use_runner_v2 \
```
Comment (zoercai) [inline run_bigquery.sh:120]: another experiment

Comment (zoercai) [reply run_bigquery.sh:120]: also add to gcs script

File: run_bigquery.sh (hunk 1 above)
Comment (zoercai) [reply run_bigquery.sh]: leave as 1

File: run_bigquery.sh (hunk 2 above)
Comment (zoercai) [reply run_bigquery.sh:120]: @thiagotnunes what was the other experiment that you wanted me to add here?

Comment (zoercai) [reply run_bigquery.sh:120]: was it `runner_harness_container_image=gcr.io/cloud-dataflow/v1beta3/unified-harness:20211003-rc00`?

Comment (thiagotnunes) [reply run_bigquery.sh:120]: We don't need that anymore

File: README.md [hunk 3]
```diff
@@ -51,3 +54,35 @@ This script will dispatch a remote job in dataflow with the specified configurat
...
+  --big-query-table-name <my-big-query-table-name> \
+  --region <my-dataflow-job-region>
+```
+
+This script will dispatch a remote job in dataflow with the specified configuration:
+
+- `-p|--project`: the Google Cloud Platform project id
+- `-i|--instance`: the Google Cloud Spanner instance id where the change stream resides
+- `-d|--database`: the Google Cloud Spanner database id where the change stream resides
+- `-mi|--metadata-instance`: the Google Cloud Spanner instance id where the Connector metadata tables will be created (we recommend it to be different than the change stream instance)
```
Comment (thiagotnunes) [inline README.md]: Could you update this to remove the `(we recommend it to ...)`. That is not necessary for the instance, only for the metadata-database.

File: pom.xml [hunk 4]
```diff
@@ -11,8 +11,8 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
 
-    <beam.version>2.32.0</beam.version>
-    <connector.version>2.33.0.1-SNAPSHOT</connector.version>
+    <beam.version>2.35.0</beam.version>
+    <connector.version>2.35.0</connector.version>
```
Comment (thiagotnunes) [inline pom.xml]: The Connector is not released yet, so we need to leave it as `2.33.0.1-SNAPSHOT`

**Review by thiagotnunes — APPROVED** [review]
After addressing the comments, feel free to merge.

**Pull request merged**